catalog/catalog-app/pom.xml

```diff
@@ -362,5 +362,15 @@
             <artifactId>catalog-plugin-metacard-validation</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ddf.catalog.transformer</groupId>
+            <artifactId>catalog-transformer-generic-impl</artifactId>
```

Comment:
This is not a generic transform api, that is already part of the catalog framework.  This is specifically a streaming xml api and impl which uses the more generic transformer api.